packages/flutter/lib/src/widgets/sliver.dart

```diff
@@ -1035,6 +1035,24 @@ class SliverList extends SliverMultiBoxAdaptorWidget {
     required super.delegate,
   });
 
+  /// Creates a sliver list from an item builder.
```

Comment:
The ListView.builder docs could probably be re-used here. There are examples of re-using docs like this if you search for `{@template...` as well as `{@macro...` in the framework.

Reply:
It might be nice to include a sample snippet of code here that shows it in use.

Reply:
Done!